canonical-web-and-design/www.ubuntu.com
# Remove /legal/managed-services section

## Done

- Remove /lega/managed-services section
- Added redirects
- Removed from the navigation
- Updated the copy docs

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal](http://0.0.0.0:8001/legal)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the /legal page has been updated per the [copy doc](https://docs.google.com/document/d/19yTLYWu7rEErPrO0xDsqMeN8tcWKBjHuarw0tjxRlhY/edit#)
- Test the redirects
    - /legal/managed-services -> /legal/ubuntu-advantage-service-description
    - /legal/managed-services-description -> /legal/ubuntu-advantage-service-description
    - /legal/managed-services-privacy -> /legal/ubuntu-advantage-service-description
    - /legal/managed-services/privacy -> /legal/ubuntu-advantage-service-description
    - /legal/managed-services/service-description -> /legal/ubuntu-advantage-service-description

## Issue / Card

Fixes #5184
